Library/Sources/SCRecorder.m

```diff
@@ -1256,6 +1272,32 @@ - (AVCaptureVideoOrientation)actualVideoOrientation {
     return videoOrientation;
 }
 
+//Uses the accelerometer data to get the orientation. Useful for when the device orientation is locked.
+- (UIDeviceOrientation)getCurrentOrientationFromAccelerometer{
+    
+    CMAccelerometerData * data = _motionManager.accelerometerData;
+    UIDeviceOrientation orientation = [[UIDevice currentDevice] orientation];
+    
+    if (data) {
+        if (fabs( data.acceleration.y ) < fabs( data.acceleration.x )) {
```

Comment:
You should follow the code style of the project if we want to get this accept upstream, e.g.

    if (fabs(data.acceleration.y) < fabs(data.acceleration.x))

and 

    } else {